src/applications/vaos/services/appointment/transformers.js

```diff
@@ -217,7 +239,57 @@ function addParticipantAndContained(appt, transformed) {
     ];
   }
 
-  return { ...transformed, participant, contained };
+  if (isCommunityCare(appt)) {
+    const contained = [];
+    const address = appt.address;
+
+    contained.push({
+      actor: {
+        name: appt.providerPractice,
+        address: {
+          line: [address?.street],
+          city: address?.city,
+          state: address?.state,
+          postalCode: address?.zipCode,
+        },
+        telecom: [
+          {
+            system: 'phone',
+            value: appt.providerPhone,
+          },
+        ],
+      },
+    });
+
+    if (!!appt.name?.firstName && !!appt.name?.lastName) {
+      contained.push({
```

Comment:
We could actually put this in the `participant` list. This is a reference, so it would make sense there.